cf-agent/verify_networks.c

```diff
@@ -415,7 +415,7 @@ static int GetArpInfo(ARPState **list, const Promise *pp)
     size_t line_size = CF_BUFSIZE;
     char *line = xmalloc(line_size);
     char comm[CF_BUFSIZE];
-    char ip[CF_MAX_IP_LEN], mac[CF_MAX_IP_LEN], device[CF_SMALLBUF], state[CF_SMALLBUF];
+    char ip[CF_MAX_IP_LEN + 1], mac[CF_MAX_IP_LEN], device[CF_SMALLBUF], state[CF_SMALLBUF];
```

Comment:
mac is filled with a maximum of 31 characters plus terminating \0, so only up to 32 bytes from the 64 byte buffer are used: the declaration allocates more memory than needed (that does not harm much).